src/liballoc/raw_vec.rs

```diff
@@ -339,14 +358,13 @@ impl<T> RawVec<T> {
             let ptr = if self.cap == 0 {
                 heap::allocate(new_alloc_size, align)
             } else {
-                heap::reallocate(self.ptr() as *mut _,
-                                 self.cap * elem_size,
-                                 new_alloc_size,
-                                 align)
+                heap::reallocate(self.ptr() as *mut _, self.cap * elem_size, new_alloc_size, align)
```

Comment:
See https://github.com/nrc/rustfmt/issues/342

I'm actually working on this right now :-)